cmd/webhook/main.go

```diff
@@ -40,6 +40,7 @@ var (
 	scheme             = runtime.NewScheme()
 	port               int
 	metricsBindAddress string
+	stopCh             = make(chan struct{})
```

Comment:
how about controllerruntime.SetupSignalHandler()? `InformerFactory` needs a stop chan

Reply:
> How about controllerruntime.SetupSignalHandler()? `InformerFactory` needs a stop chan

Yeah, maybe `controllerruntime.SetupSignalHandler()` is a better way to do this, rather than doing some extra channel-close logics.

Reply:
Why we need a informer in webhook? I prefer what you want is a cacher not a controller

Reply:
BTW, controller-runtime provide us some useful utility packages about this, try them!

Reply:
> Why we need a informer in webhook? I prefer what you want is a cacher not a controller

Yes, I need a cache. Are there any better solution?

Reply:
"Informer in webhook" problem has another conversation, this will be set resolved.

Reply:
"Informer in webhook" is a later considering.